src/spec-configuration/httpOCIRegistry.ts

```diff
@@ -186,6 +194,20 @@ async function getCredential(params: CommonParams, ociRef: OCIRef | OCICollectio
 							refreshToken: undefined,
 						};
 					}
+					if (dockerConfig.credHelpers && dockerConfig.credHelpers[registry]) {
+						const credHelper = dockerConfig.credHelpers[registry];
+						output.write(`[httpOci] Found credential helper '${credHelper}' in config.json for registry '${registry}'`, LogLevel.Trace);
+						const auth = await getCredentialFromHelper(params, registry, credHelper);
+						if (auth) {
+							return auth;
+						}
+					} else if (dockerConfig.credsStore) {
```

Comment:
Let's say the user has credHelper but the CLI wasn't able to get auth from it, then shouldn't we also check for credsStore?
Hence, should this be an `if` condition as well?

Reply:
The Docker docs say:

> The default credential store (`credsStore` or the config file itself) will not be used for operations concerning credentials of the specified registries.

https://docs.docker.com/engine/reference/commandline/login/

I think this means that if a registry has a `credHelpers` entry, `credsStore` should not be used for that registry.